text/0000-fabric-private-chaincode-lite.md

```diff
@@ -247,27 +241,38 @@ This class includes smart contracts which operate on sensitive medical data, or
 The former is illustrated in more details below.
 More detailed information on the constraints and related programming model restrictions plus a corresponding security analysis can be found in a [separate document](https://docs.google.com/document/d/1jbiOY6Eq7OLpM_s3nb-4X4AJXROgfRHOrNLQDLxVnsc/).
 
-The [Rollback Protection Extension](#rollback-protection-extension) to FPC Lite can enable support of an even larger class of applications such as private sealed-bid auctions or secure voting which do have intrinsic requirements of release of sensitive data conditioned on private state, e.g., a sealed bid auction outcome should be private until the auction closure is commited on the ledger.
+<!-- The [Rollback Protection Extension](#rollback-protection-extension) to FPC Lite can enable support of an even larger class of applications such as private sealed-bid auctions or secure voting which do have intrinsic requirements of release of sensitive data conditioned on private state, e.g., a sealed bid auction outcome should be private until the auction closure is committed on the ledger. -->
```

Comment:
I would leave this: this helps more clearly defining the class of apps we support ...

Reply:
this is part of the last paragraph in the `Use case: Privacy-enhanced Federated Learning on FPC Lite`